catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -442,13 +444,23 @@ const WorkDetails = ({
                 ? `View ${work.images.length} images`
                 : 'View 1 image'
             }
-            link={worksLink(
-              {
-                search: 'images',
-                query: work.id,
-              },
-              'work_details/images'
-            )}
+            link={
+              searchPrototype
+                ? imagesLink(
+                    {
```

Comment:
wrap the object in const and can be used for both as its same content.

Reply:
I was thinking about that, but given this is code that will necessarily be reverted at some point, it felt simpler to keep a bit of duplication in the template where all the changes can be made at once at the expense of being DRY.